data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,23 @@
 [
   {
+    "id": "cooking_standard",
+    "type": "requirement",
+    "//": "Cooking of standard meals",
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [
+      [ "fire", -1 ],
+      [ "hotplate", 10 ],
+      [ "mess_kit", 10 ],
+      [ "mil_mess_kit", 10 ],
+      [ "oil_cooker", 10 ],
+      [ "survivor_mess_kit", 10 ],
+      [ "gasoline_cooker", 10 ],
+      [ "charcoal_cooker", 10 ],
+      [ "popcan_stove", 10 ],
+      [ "toolset", 10 ]
+    ] ]
+  },
```

Comment:
> The rationale here is that for example we could scale mil_mess_kit to be more efficient than mess_kit and so on?

Specifically for heat sources, the efficiency shouldn't increase, only decrease from the baseline. Conversion from electricity to heat is nearly perfect.
At the very least, the multiplier next to hotplates should be 1. Some other cookers can be worse. Most notably the oil cookers - 4 tainted fats (about 1.25 of a butchered fat zed) is 250 charges of lamp oil.

> The other obvious base units are 2, 4 and 5. I don't want to use 1 as this precludes any later tools with higher efficiency.

4-5 would be tolerable. I can't think of a recipe that should use less than that. 10 would drop the granularity too low and would force using regular fires.
Until we have a proper requirement for "keep fire alive through the whole crafting process", the annoying fire exploit thing has to be taken into consideration. That is, if heat recipes that allow fire become too expensive, the exploit becomes rewarding. Annoying, tedious things shouldn't be rewarding and optimal.